pkg/controlplane/apiserver/options/validation_test.go

```diff
@@ -56,12 +56,20 @@ func TestValidateAPIPriorityAndFairness(t *testing.T) {
 		},
 		{
 			runtimeConfig:    "flowcontrol.apiserver.k8s.io/v1beta3=false",
-			errShouldContain: conflict,
+			errShouldContain: "",
```

Comment:
if we don't server a version (v1beta3), it can still be a storage version, right?

Reply:
Yes